BlazorUI/Data/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace BlazorUI.Data
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assign this magic number '0.5556' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)